### PR TITLE
Bug - Edit delete icon is not showing for a few timesheet entries 

### PR DESCRIPTION
**Pull request by abinashpa**
## Notion card
https://www.notion.so/saeloun/not-getting-edit-and-delete-icons-for-a-few-of-the-timesheet-entries-d22e2ff6b74f450dacbce73d0db15d7f

## Summary
- Edit and delete buttons in /time-tracking were not showing in some timesheet entries.

## Preview
https://www.loom.com/share/8324c00a83cd4a27b86b85c7492b13a7

## Type of change
- Moved icons (edit and delete) to app/assets.

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
- Tested on my mac by creating multiple timesheet entries and hovering over the cards where edit and delete.

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/controllers/internal_api/v1/projects_controller.rb [hunk 1]
```diff
@@ -28,7 +28,12 @@ def update
 
   def destroy
     authorize project
-    project.discard!
+    projects = project.client.projects
```
Comment (keshavbiswa) [inline app/controllers/internal_api/v1/projects_controller.rb:31]: Please ensure you have added inverse_of associations to project has_many clients associations.

File: app/javascript/src/components/Profile/UserDetail/index.tsx [hunk 2]
```diff
@@ -223,7 +223,7 @@ const UserDetails = () => {
                   </input>
                   <button className="" onClick={handleDeleteLogo}>
                     <img
-                      src="/delete.svg"
+                      src={dele}
```
Comment (keshavbiswa) [inline app/javascript/src/components/Profile/UserDetail/index.tsx:226]: Is this correct, I'm wondering?

File: app/controllers/internal_api/v1/projects_controller.rb (hunk 1 above)
Comment (keshavbiswa) [reply app/controllers/internal_api/v1/projects_controller.rb:31]: https://guides.rubyonrails.org/association_basics.html#bi-directional-associations

**Comment by apoorv1316** [comment]
@abinashpa Can you please reply to the PR reviewers before marking the conversation as resolved?

**Comment by abinashpa** [comment]
> @abinashpa Can you please reply to the PR reviewers before marking the conversation as resolved? Also Why there are 3 months old commits on this PR?

In abhove comments it's wasn't necessary because there is only a typo other changes have been removed and a rails guide link.

**Review by keshavbiswa — APPROVED** [review]
LGTM!

**Comment by supriya3105** [comment]
@apoorv1316 Lets review this. Please check notion ticket for bug description when you test.

**Comment by apoorv1316** [comment]
> @apoorv1316 Lets review this. Please check notion ticket for bug description when you test.

@supriya3105 Yes let's do it in standup. Haven't reviewed it because wanted to see the entries first where the buttons were not visible.

File: app/javascript/src/components/time-tracking/WeeklyEntriesCard.tsx [hunk 3]
```diff
@@ -206,13 +209,13 @@ const WeeklyEntriesCard = ({
...
             alt="edit"
             className="icon-hover ml-8 cursor-pointer"
           />
           {/* <img
             onClick={handleDeleteEntries}
-            src="/delete.svg"
+            src={deleteIcon}
             alt="delete"
             className="icon-hover ml-8 cursor-pointer"
           /> */}
```
[the remark this answers is not in the document]
Comment (abinashpa) [reply app/javascript/src/components/time-tracking/WeeklyEntriesCard.tsx:221]: Currently, we have disabled the edit and delete buttons for weekly entries.

File: app/javascript/src/components/time-tracking/WeeklyEntriesCard.tsx [hunk 4]
```diff
@@ -10,6 +10,9 @@ import validateTimesheetEntry from "helpers/validateTimesheetEntry";
 
 const checkedIcon = require("../../../../assets/images/checkbox-checked.svg");
 const uncheckedIcon = require("../../../../assets/images/checkbox-unchecked.svg");
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
[the remark this answers is not in the document]
Comment (abinashpa) [reply app/javascript/src/components/time-tracking/WeeklyEntriesCard.tsx]: Removed.

**Comment by supriya3105** [comment]
@apoorv1316 Can you test this locally .

**Comment by apoorv1316** [comment]
@supriya3105 Please see this loom video https://www.loom.com/share/30b2bd4338f54c7480f1e541ef16ce8c

**Comment by supriya3105** [comment]
> @supriya3105 Please see this loom video https://www.loom.com/share/30b2bd4338f54c7480f1e541ef16ce8c

@apoorv1316 The video does not play after 7 seconds. So unable to watch it.

**Comment by apoorv1316** [comment]
> https://www.loom.com/share/30b2bd4338f54c7480f1e541ef16ce8c



> > @supriya3105 Please see this loom video https://www.loom.com/share/30b2bd4338f54c7480f1e541ef16ce8c
> 
> @apoorv1316 The video does not play after 7 seconds. So unable to watch it.

Weird. I can play it even in incognito. Can you try that too?

**Comment by supriya3105** [comment]
@apoorv1316 Watched the video. 

I could not understand the scroll issue you mentioned. Lets discuss after updates on standup call. 

Regarding scenarios- Can you add 4-5 tasks for each day of a week and test that edit and delete icon are visible.

**Comment by apoorv1316** [comment]
@supriya3105 It looks good to me even if there are multiple long entries https://www.loom.com/share/f1da9237f12445528a76850fd4e2d2b9

**Review by apoorv1316 — APPROVED** [review]

**Comment by JijoBose** [comment]
I had the same issue today when you create an entry and enter the description with more than 100 characters without a line break. 
Notice that the edit and delete button gets smaller.

**Review by akhilgkrishnan — APPROVED** [review]
LGTM

**Pull request merged**